supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -81,18 +84,21 @@ class JesInitializationActor(jesParams: JesInitializationActorParams)
   } yield JesBackendInitializationData(jesWorkflowPaths, runtimeAttributesBuilder, jesConfiguration, gcsCreds, genomicsFactory)
 
   override def beforeAll(): Future[Option[BackendInitializationData]] = {
-    def fileUpload(paths: JesWorkflowPaths): Future[Unit] = {
-      writeAuthenticationFile(paths, jesConfiguration.jesAttributes.restrictMetadataAccess, jesConfiguration.dockerCredentials) recoverWith {
-        case CromwellFatalException(e: StorageException) if e.getCode == StatusCodes.PreconditionFailed.intValue && jesParams.restarting =>
-          workflowLogger.debug(s"Authentication file already exists but this is a restart, proceeding.")
-          Future.successful(())
-        case CromwellFatalException(e: StorageException) if e.getCode == StatusCodes.PreconditionFailed.intValue =>
-            Future.failed(new IOException(s"Failed to upload authentication file:" +
-              s" there was already a file at the same location and this workflow was not being restarted: ${e.getMessage}"))
-        case failure => Future.failed(new IOException(s"Failed to upload authentication file", failure))
-      }
+    def fileUpload(paths: JesWorkflowPaths) = existsAsync(paths.gcsAuthFilePath) flatMap {
+      // if it is then something is definitely wrong
+      case true if !jesParams.restarting =>
+        Future.failed(AuthFileAlreadyExistsException(paths.gcsAuthFilePath))
+      case true => 
+        workflowLogger.debug(s"Authentication file already exists but this is a restart, proceeding.")
+        Future.successful(())
```

Comment:
ToL: What if it was partially uploaded? Can we do a sanity check on the contents?

Reply:
I guess we could, I kind of want to leave that to another ticket though as I'm not trying to fix everything wrong with initialization but to fix what causes travis to fail now.

Reply:
Also I know the cloud is magical and I shouldn't make assumptions about how anything is implemented but I'd be pretty surprised if we had an issue with a partially uploaded auth file.